src/Announcements-Core/Announcer.class.st

```diff
@@ -27,16 +27,22 @@ Class {
 Announcer >> announce: anAnnouncement [
 
 	^ self isSuspended
-		  ifFalse: [
-			  | announcement |
-			  announcement := anAnnouncement asAnnouncement.
-
-			  "If the user required to prevent the current announcement, we stop here."
-			  (self preventedAnnouncements handlesAnnouncement: announcement) ifTrue: [ ^ self ].
-
-			  registry ifNotNil: [ registry deliver: announcement ].
```

Comment:
I removed the ifNotNil: check because it's initialized on creation, it's never nil